presto-main/src/test/java/io/prestosql/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -182,6 +194,13 @@ private PlanMatchPattern addColumnReferences(String expectedTableName, Map<Strin
         return this;
     }
 
+    private PlanMatchPattern addColumnReferencesWithHandles(Map<String, Predicate<ColumnHandle>> expectedColumns)
```

Comment:
This is only used in one place and doesn't contribute to readability, so I'd inline it.